homeassistant/components/vizio/config_flow.py

```diff
@@ -30,7 +30,8 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def _config_flow_schema(input_dict: Dict[str, Any]) -> vol.Schema:
+# pylint: disable=dangerous-default-value
```

Comment:
We shouldn't pass mutable objects as default values.

Reply:
Will revert